Python/wolfssl-python-3.8.14.patch

```diff
@@ -1188,45 +1189,41 @@ index 71cfdcd..937a15a 100644
          if "ADH" not in parts and "EDH" not in parts and "DHE" not in parts:
              self.fail("Non-DH cipher: " + cipher[0])
  
-@@ -4258,7 +4469,13 @@ class ThreadedTests(unittest.TestCase):
+@@ -4258,7 +4480,10 @@ class ThreadedTests(unittest.TestCase):
              stats = server_params_test(client_context, server_context,
                                         chatty=False,
                                         sni_name='supermessage')
 -        self.assertEqual(cm.exception.reason, 'TLSV1_ALERT_ACCESS_DENIED')
 +        if ssl.IS_WOLFSSL:
-+            # Depending on threading and socket shutdown when test is run,
-+            # reason may occasionally be None here
-+            if cm.exception.reason is not None:
-+                self.assertEqual(cm.exception.reason, 'fatal error')
++            self.assertEqual(cm.exception.reason, 'ALERT_FATAL_ERROR')
```

Comment:
This change might lead to flaky test results. The change is okay as is, mainly making the note to help remind myself to keep an eye on retry rates.

Reply:
@JacobBarthelmeh this change was already here. It was previously appended to the end of the original patch. It took Aidan and I a while to figure that out. This is the state of the current patch.

Thank you for merging. Let's see if the Python tests continue passing and I'll retry #7771 and see if it passes now.